ipn/ipnlocal/local.go

```diff
@@ -2735,6 +2735,13 @@ func (b *LocalBackend) CheckIPNConnectionAllowed(ci *ipnauth.ConnIdentity) error
 	if !b.pm.CurrentPrefs().ForceDaemon() {
 		return nil
 	}
+
+	// Always allow Windows SYSTEM user to connect,
+	// even if Tailscale is currently being used by another user.
+	if tok, err := ci.WindowsToken(); err == nil && tok.IsSystemUser() {
```

Comment:
Please `Close` `tok` after checking. Internally it has a finalizer, but there's no sense keeping that handle open indefinitely.

Reply:
done